examples/megaclc/megaclc.cpp

```diff
@@ -83,6 +83,21 @@ using m::logDebug;
 #endif
 #endif
 
+bool g_detailHigh = false;
+
+std::atomic<bool> g_reportMessagesDeveloper{false};
+
+// These objects are helping to work around history loading problems for reviewing public chats
+std::atomic<bool> g_reviewingPublicChat{false};
+std::atomic<int> g_reviewPublicChatMsgCountRemaining{0};
+std::unique_ptr<std::ofstream> g_reviewPublicChatOutFile;
+std::unique_ptr<std::ofstream> g_reviewPublicChatOutFileLinks;
+std::mutex g_reviewPublicChatOutFileLogsMutex;
+std::unique_ptr<std::ofstream> g_reviewPublicChatOutFileLogs;
+std::map<c::MegaChatHandle, std::string> g_reviewPublicChatEmails;
+std::map<c::MegaChatHandle, std::string> g_reviewPublicChatFirstnames;
+std::map<c::MegaChatHandle, std::string> g_reviewPublicChatLastnames;
```

Comment:
I moved this up so that the new file handle and mutex are available in the logger classes.